### DISCUSSION TITLE
How to add set or max height to table rows?

### DISCUSSION THREAD

**Discussion by KyleBrown-804** [element-plus/element-plus#16280] [Q&A]
Basically what the title says, I have tried to for example set a max height of 20px for table rows, but in a couple things I've tried I have not had success, I've tried passing via table `row-style` and `cell-style` props with height and max-height 20px. I've also tried from the top level overriding the elements CSS to enforce a set height but no success so far.

I also don't see any row height or similar props listed under Table API but maybe I'm just missing something?: https://element-plus.org/en-US/component/table.html#table-api.

**Comment by Aseisman** [comment]
because the row 's display is table-row;  you can change the inline dom‘s style and set the max-height
